MN/Algoritmi/Algoritmi-ad-hoc/IsPositiveDefinite.m

```diff
@@ -0,0 +1,16 @@
+% checks if a matrix A is positive definite using Sylvester's theorem
```

Comment:
"checks if a matrix A is positive-definite"
atat. se numeste IsPositiveDefinite. nu trebuie nimeni sa stie din exterior cum face asta. si oricum trebuie sa incerci si diag dominanta, nu numai Sylvester.